.coveragerc

```diff
@@ -0,0 +1,7 @@
+[run]
+omit =
+    /*/tests/*
+    /*/clang/*
+
+[report]
+ignore_errors = True
```

Comment:
nit: `ensure_newline_at_eof_on_save: true` option is generally recommended in coding.